### ISSUE TITLE
[Feature Request] Make block list searchable

### ISSUE THREAD

**Issue by tsievert** [diegoberaldin/RaccoonForLemmy#1153]
**Is your feature request related to a problem? Please describe.**
I recently needed to go through my community block list to check if I had accidentally blocked a few communities. 
It was extremely tedious to do via the app, as I could not just search for the community name, but had to scroll through the entire list.

In the end I used my browser to check my block list, and there I could just CTRL+F to the community I was looking for and remove it.

**Comment by diegoberaldin** [comment]
Yes, this is a great idea! The API does not allow filtering but I can do it client-side (since there is no pagination involved).
